src/components/screens/EventRecord.js

```diff
@@ -182,6 +184,22 @@ export const EventRecord = ({ data, route }) => {
           title={texts.eventRecord.operatingCompany}
         />
 
+        {settings?.displayOnlySummary === 'true' && !!settings?.onlySummaryLinkText && (
+          <Wrapper>
+            {webUrls.map((item, index) => {
+              const { url } = item;
+
+              return (
+                <Button
+                  key={index}
+                  title={settings.onlySummaryLinkText}
+                  onPress={() => openLink(url, openWebScreen)}
+                />
+              );
+            })}
```

Comment:
Here we only use the `url` information from the item and there is no control inside the `Button` component. How about using it as `webUrls.map(({ url }, index) => {...}` ?
for example:

```
{webUrls.map(({ url }, index) => (
  <Button
    key={index}
    title={settings.onlySummaryLinkText}
    onPress={() => openLink(url, openWebScreen)}
  />
))}
```

Reply:
good idea! i like the simplicity about this.

Reply:
8987358